imagepuller/internal/service/verify_test.go

```diff
@@ -12,17 +14,25 @@ import (
 
 	"github.com/containers/storage"
 	"github.com/edgelesssys/contrast/imagepuller/internal/test/registry"
+	"github.com/google/go-containerregistry/pkg/name"
+	gcr "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/opencontainers/go-digest"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 type StubStore struct {
 	putLayerLayer digest.Digest
+	putLayerErr   bool
+
 	storage.Store
 }
 
 func (s *StubStore) PutLayer(_, _ string, _ []string, _ string, _ bool, _ *storage.LayerOptions, _ io.Reader) (*storage.Layer, int64, error) {
+	if s.putLayerErr {
+		return nil, 0, errors.New("")
```

Comment:
[`assert.AnError`](https://pkg.go.dev/github.com/stretchr/testify/assert#AnError) is useful here.